HW_5/task_3.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задача 3.
+Создайте функцию генератор чисел Фибоначчи (см. Википедию).
+"""
+from time import sleep
+
+
+def fibonacci_gen():
+    x = 0
+    y = 1
+    while True:
+        yield x + y
+        sleep(2)
+        y = x + y
+        x = y - x
```

Comment:
Можно сделать проще.
```suggestion
        yield x
        sleep(2)
        x, y= y, x + y
```